Documentation/ceph-cluster-crd.md

```diff
@@ -1075,15 +1075,15 @@ spec:
 
 ### Cleanup policy
 
-Rook has the ability to cleanup resources and data that were deployed.
+Rook has the ability to cleanup resources and data that were deployed when a `delete cephcluster` command is issued.
 The policy represents the confirmation that cluster data should be forcibly deleted.
 The cleanupPolicy should only be added to the cluster when the cluster is about to be deleted.
 After the `confirmation` field of the cleanup policy is set, Rook will stop configuring the cluster as if the cluster is about to be destroyed in order to prevent these settings from being deployed unintentionally.
 The `cleanupPolicy` CR settings has different fields:
 
-* `confirmation`: If `yes-really-destroy-data` the operator will automatically delete data on the hostpath of cluster nodes and clean devices with OSDs when a `delete cephcluster` command is issued. Only `yes-really-destroy-data` and an empty string are valid values for this field.
+* `confirmation`: Only an empty string and `yes-really-destroy-data` are valid values for this field. If an empty string is set, Rook will only remove Ceph's metadata, which allows a cluster re-installation if desired. If `yes-really-destroy-data` the operator will automatically delete data on the hostpath of cluster nodes and clean devices with OSDs.
```

Comment:
To clarify, re-install is not possible if this setting is empty without an additional step.
```suggestion
* `confirmation`: Only an empty string and `yes-really-destroy-data` are valid values for this field. If an empty string is set, Rook will only remove Ceph's metadata. A re-installation will not be possible unless the hosts are cleaned first. If `yes-really-destroy-data` the operator will automatically delete data on the hostpath of cluster nodes and clean devices with OSDs. The cluster can then be re-installed if desired with no further steps.
```

Reply:
Thanks. fixed